variablegroups/PERTTEST-AKS-Common.json

```diff
@@ -0,0 +1,71 @@
+{
+  "variables": {
+    "Env.Core.ResourceGroup": {
+      "value": "core-infra-$(Env.Override.Name)-rg"
+    },
+    "Env.Override.Name": {
+      "value": "perftest"
+    },
+    "Environment.name": {
+      "value": "perftest"
+    },
+    "Subscription.acrName": {
+      "value": "hmctspublic"
+    },
+    "Subscription.AKS.Appgw.ingress": {
+      "value": "[{ \"IpAddress\": \"10.10.41.250\" }, { \"IpAddress\": \"10.10.45.250\" }]"
+    },
+    "Subscription.AKS.Appgw.subnet.cidr": {
+      "value": "10.10.46.0/25"
+    },
+    "Subscription.AKS.Environment.Name": {
+      "value": "perftest"
+    },
+    "Subscription.AKS.Helm.TLS": {
+      "value": "true"
+    },
+    "Subscription.AKS.Iass.subnet.cidr": {
+      "value": "10.10.47.0/24"
+    },
+    "Subscription.AKS.Kubernetes.version": {
+      "value": "1.13.5"
+    },
+    "Subscription.AKS.nodeCount": {
+      "value": "3"
+    },
+    "Subscription.AKS.Subnet.Primary.Cidr": {
+      "value": "10.10.40.0/23"
+    },
+    "Subscription.AKS.Subnet.Secondary.Cidr": {
+      "value": "10.10.44.0/23"
+    },
+    "Subscription.AKS.vmSize": {
+      "value": "Standard_D4s_v3"
+    },
+    "Subscription.AKS.VNet.Cidr": {
+      "value": "10.10.40.0/21"
+    },
+    "Subscription.AKS.VNet.DnsServers": {
+      "value": "[\"10.96.139.254\"]"
```

Comment:
This one seems to be of AAT ? Not sure if we use same for perftest , worth checking.

Reply:
yeah updated the one for perftest.